codex-ui/dev/pages/components/ContextMenu.vue

```diff
@@ -9,6 +9,7 @@
   <ContextMenu
     :show-search="true"
     :items="contextMenuItems"
+    :message="'Nothing found'"
```

Comment:
unclear property name

Reply:
I think we can get rid of it at all since it is a static text, you can pass it inside the ContextMenu to item